src/pages/enum/EnumDeclaration.sol

```diff
@@ -1,4 +1,5 @@
 // SPDX-License-Identifier: MIT
+// This is saved 'EnumDeclaration.sol'
```

Comment:
Please put this comment on line 3 after compiler version.
It's common practice to have license on first line, compiler version on second line.

Reply:
Fixed 👍